### PR TITLE
Resolve known issues for policies dashboards

### PR DESCRIPTION
**Pull request by dislbenn**

### CODE REVIEW

**Review by clyang82 — APPROVED** [review]
LGTM

File: operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-policy-group-compliancy-overview.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ data:
       "panels": [
         {
           "datasource": "${datasource}",
-          "description": "Aggregated policy group trend by: standard - $standard, category - $category, control - $control, and severity - $severity",
+          "description": "Aggregated policy group trend by standard, category, control, and severity.",
```
Comment (dislbenn) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-policy-group-compliancy-overview.yaml:45]: Jira issue: [ACM-5524](https://issues.redhat.com/browse/ACM-5524)

**Comment by dislbenn** [comment]
@clyang82 please help to review again.

**Review by clyang82 — APPROVED** [review]
LGTM

**Comment by yanmxa** [comment]
@clyang82 This unit test error is caused by the conflict of the global hub agent metrics server. It also appears in this PR https://github.com/stolostron/multicluster-global-hub/pull/449, and the error also be fixed in it.
So you can rebase this PR once the https://github.com/stolostron/multicluster-global-hub/pull/449 is merged.

**Pull request merged**